src/Sarif/ExtensionMethods.cs

```diff
@@ -16,6 +16,11 @@ namespace Microsoft.CodeAnalysis.Sarif
 {
     public static class ExtensionMethods
     {
+        public static bool HasAtLeastOneNonNullValue(this IList<string> list)
+        {
+            return list != null && list.Where((s) => s != null).FirstOrDefault() != null;
```

Comment:
>list.Where((s) => s != null).FirstOrDefault() != null; [](start = 35, length = 54)

You can simplify this because `Any` can take a predicate:
```
return list != null && list.Any(s => s != null);
```

Reply:
like it, thanks.

---
In reply to: [229079586](https://github.com/Microsoft/sarif-sdk/pull/1121#discussion_r229079586) [](ancestors = 229079586)